polyphemus/lib/etls/add_watch_folder_base_etl.rb

```diff
@@ -140,7 +140,6 @@ def folders_files(cursor, folders)
     find_request = Etna::Clients::Metis::FindRequest.new(
       project_name: cursor[:project_name],
       bucket_name: cursor[:bucket_name],
-      hide_paths: true,
```

Comment:
I think these changes are required to allow the matrix / attribute processors to work. They have to download files, which means you can't construct the download URL by hand from a known folder_path (has to be signed by Metis).

Reply:
ahhh shoot.  ok, sure.  this is fine.  I totally missed this while testing yesterday because I was cofused on the linker.